README.md

```diff
@@ -112,8 +112,13 @@ Pass the product and customer identifier to the following function when your cus
 In case this parameter is not passed, then the **customerId** will be the same as the **SubscriptionId** created in Chargebee.
 
 ```ts
-import Chargebee, { Purchase } from '@chargebee/react-native-chargebee';
-
+import Chargebee, { Purchase, Customer } from '@chargebee/react-native-chargebee';
```

Comment:
Nit: Should `Purchase` and `Customer` be imported as types?

Reply:
I believe this would be dependent on the typescript version used in the app.